en_US/changes/changes-4.3.md

```diff
@@ -15,6 +15,91 @@ ref: undefined
 
 # Changes
 
+## 4.3.0 版本
+
+*Release Date: 2021-05-06*
+
+EMQ X 4.3.0 is released now, it mainly includes the following changes:
+
+### Features and Enhancement
+
```

Comment:
Add 2 non-compatible changes:

1.  emqx_auth_client and emqx_auth_username are merged to emqx_auth_mnesia
Please refer to doc (TODO: link) to export data from older versions, and import to v4.3

2. By default, docker only logs to console, set EMQX_LOG__TO=file to switch to file